cypress.config.js

```diff
@@ -1,18 +1,33 @@
-const { defineConfig } = require("cypress");
-const faker = require('faker');
+const { defineConfig } = require('cypress');
+const faker = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
     baseUrl: 'https://conduit.mate.academy/',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
-          const email = faker.internet.email();
-          const randomNumber = Math.floor(Math.random(1000) * 1000);
+          const username = faker.person.name();
+          const email = faker.internet.email().toLowerCase();
+          const password = '12345Qwert!';
           return {
-            username: faker.name.firstName() + randomNumber,
-            email: email.toLowerCase(),
-            password: '12345Qwert!'
+            username,
+            email,
+            password
+          };
+        }
+      });
+      on('task', {
+        generateDataForArticle() {
+          const title = faker.lorem.words(3);
+          const description = faker.lorem.sentence();
+          const body = faker.lorem.paragraph();
+          const tagList = [faker.lorem.word()];
+          return {
+            title,
+            description,
+            body,
+            tagList
           };
         }
       });
```

Comment:
You have duplicated the `on('task', ...)` event. You can define multiple tasks within a single `on('task', ...)` event. I recommend that you combine the two `on('task', ...)` events into one for better readability and performance.